### PR TITLE
fix(kit): change min/max from Infinity to min/max integers

### PR DESCRIPTION
**Pull request by zarghamkhandev**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # https://github.com/Tinkoff/taiga-ui/issues/2421

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by zarghamkhandev** [comment]
Do we also want to limit the input to the component ? So that it does not exceed MAX/MIN safer integers ? I only changed the default values. @waterplea @MarsiBarsi

**Comment by waterplea** [comment]
> Do we also want to limit the input to the component ? So that it does not exceed MAX/MIN safer integers ? I only changed the default values. @waterplea @MarsiBarsi 

I don't think that extra safety is necessary.

File: projects/kit/components/input-number/input-number.component.ts [hunk 1]
```diff
@@ -70,11 +70,11 @@ export class TuiInputNumberComponent
 
     @Input()
     @tuiDefaultProp()
-    min = -Infinity;
+    min = Number.MIN_SAFE_INTEGER;
```
Comment (splincode) [inline projects/kit/components/input-number/input-number.component.ts:73]: What do you think, will you be able to add a unit test or a demo example with set max value inside for check on e2e?

**Review by splincode — APPROVED** [review]

File: projects/kit/components/input-number/input-number.component.ts (hunk 1 above)
Comment (zarghamkhandev) [reply projects/kit/components/input-number/input-number.component.ts:73]: sure, will do that.

**Review by waterplea — APPROVED** [review]

**Pull request closed without merging**